src/main/java/com/starkbank/DynamicBrcode.java

```diff
@@ -0,0 +1,373 @@
+package com.starkbank;
+
+import com.starkbank.utils.Generator;
+import com.starkbank.utils.Resource;
+import com.starkbank.utils.Rest;
+import com.starkbank.utils.SubResource;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+
+public final class DynamicBrcode extends Resource {
+    /**
+     * DynamicBrcode object
+     * <p>
+     * When you initialize a DynamicBrcode, the entity will not be automatically
+     * sent to the Stark Bank API. The 'create' function sends the objects
+     * to the Stark Bank API and returns the list of created objects.
+     *
+     * DynamicBrcodes are conciliated BR Codes that can be used to receive Pix transactions in a convenient way.
```

Comment:
Mesma dúvida que eu tive no de PHP.

Não seria?

When a DynamicBrcode is paid, a Deposit is created with the tags parameter containing the character “d” followed by the DynamicBrcode’s uuid "d{uuid}" for conciliation.
Additionally, all tags passed on the DynamicBrcode will be transferred to the respective Deposit resource.

Reply:
no python tava errado :P